samples/servicecontrol/retry-messages/sample.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Retry failed messages using ServicePulse
+summary: A sample showing how to retry failed messages
+component: Core
+reviewed: 2024-09-03
+---
+
+This sample shows how to retry a failed message. The sample uses the [Learning Transport](/transports/learning/) and a portable version of the Particular Service Platform tools. Installing ServiceControl is **not** required.
+
+include: platformlauncher-windows-required
+
+downloadbutton
+
+## Running the project
+
+Running the project will result in 3 console windows. Wait a moment until the ServicePulse window opens in the browser.
+
+![service-pulse-fresh](service-pulse-fresh.png)
```

Comment:
Images of ServicePulse are much to wide.

Reply:
Replaced.